root/io/stdarray/CMakeLists.txt

```diff
@@ -72,7 +72,8 @@ ROOTTEST_ADD_TEST(modelReadDict2TFile
                   MACRO  modelReadDict2.C
                   MACROARG "\"model.root\""
                   OUTREF modelReadDict2.ref
-                  DEPENDS aclicModelWrite)
+                  DEPENDS aclicModelWrite
+                  POSTCMD ${CMAKE_COMMAND} -E remove modelForReadDict2_cpp.rootmap)
```

Comment:
That's dangerous and/or will lead to confusion.  If the test requires the modelForReadDict2_cpp.rootmap file and it is run a second time, it will (mysteriously) fail the 2nd time (since the library wont be rebuilt).